gax/src/main/java/com/google/api/gax/batching/BatcherImpl.java

```diff
@@ -102,15 +107,54 @@ public BatcherImpl(
       BatchingSettings batchingSettings,
       ScheduledExecutorService executor) {
 
+    this(batchingDescriptor, unaryCallable, prototype, batchingSettings, executor, null);
+  }
+
+  /**
+   * @param batchingDescriptor a {@link BatchingDescriptor} for transforming individual elements
+   *     into wrappers request and response
+   * @param unaryCallable a {@link UnaryCallable} object
+   * @param prototype a {@link RequestT} object
+   * @param batchingSettings a {@link BatchingSettings} with configuration of thresholds
+   * @param flowController a {@link FlowController} for throttling requests. If it's null, create a
+   *     {@link FlowController} object from {@link BatchingSettings#getFlowControlSettings()}.
```

Comment:
More like a question: above the periods are removed, which I assume is the desired formatting. Here the period is kept in tact - is it because it was forgotten, or do we need a period if the description consists of multiple sentences (like here)?

Reply:
The Orcle javadoc guidelines cover this explicitly. This is correct.